llvm/test/Feature/Repo/repo_alias_aliasOfAlias.cpp

```diff
@@ -0,0 +1,21 @@
+// This test is Linux-only.
```

Comment:
Does this mean Linux host or Linux target? I think that there is a requires for hosts (e.g. REQUIRES: system-linux)

Reply:
FWIW, I think this requires a target for which clang uses alias ctors (Linux being an example). Rewriting the test to use IR should avoid this and enable the test to be run everywhere.

Reply:
> 
> 
> Does this mean Linux host or Linux target? I think that there is a requires for hosts (e.g. REQUIRES: system-linux)

Sorry, mistakenly added the comment. The test should be run everywhere.

> 
> 
> FWIW, I think this requires a target for which clang uses alias ctors (Linux being an example). Rewriting the test to use IR should avoid this and enable the test to be run everywhere.

Yes, I have rewritten a IR test. Thanks.